dockerenforcer/killer.py

```diff
@@ -38,7 +40,9 @@ def register_killed(self, container, reason):
             if container.cid in self.__killed_containers.keys():
                 self.__killed_containers[container.cid].record_new(reason)
             else:
-                self.__killed_containers[container.cid] = Stat(container.params['Name'], reason)
+                self.__killed_containers[container.cid] = Stat(
+                    container.params['Name'], reason, container.params['Config']['Image'], container.params['Config']['Labels']
```

Comment:
@piontec I agree. We can add at least a parameter `labels` to `record_new` method. I am nut sure should be add `image` to this method as it can not be change for the same containers but just in case we can add it